app/assets/javascripts/beak/widgets/ractives/edit-form.coffee

```diff
@@ -6,12 +6,17 @@ window.EditForm = Ractive.extend({
   view:         undefined # Element
 
   data: -> {
-    amProvingMyself: false     # Boolean
-  , idBasis:         undefined # String
-  , style:           undefined # String
-  , visible:         undefined # Boolean
-  , xLoc:            undefined # Number
-  , yLoc:            undefined # Number
+    parentClass:     'netlogo-widget-container' # String
+  , submitLabel:     'OK'       # String
+  , cancelLabel:     'Cancel'   # String
+  , horizontalOffset: undefined # Number
+  , verticalOffset:   undefined # Number
```

Comment:
`verticalOffset` is set on the `blockEditForm` in `RactiveNetTangoSpaces`.  `horizontalOffset` was then included since it seemed logical to have the partner value available, too.  `cancelLabel` for the same reason.